README.md

```diff
@@ -0,0 +1,69 @@
+# Open Telemetry + GoLang StatsD Demo
+
+This project demonstrates a simple Go application with StatsD metrics and Open Telemetry Collector,
+all orchestrated using Docker Compose. The application generates its own load and reports metrics, which are
+then collected by OTel Collector and sent to a Oodle endpoint.
+
+## Prerequisites
+
+- Docker
+- Docker Compose
+- The application requires 6767, 8125 and 8888 ports to be free.
```

Comment:
4318 as well, right?

Reply:
No, removed 4318 from otel collector. That port is to be used for otlp traffic which we don't have in this app.